lib/Resources/config/internal.yml

```diff
@@ -51,6 +51,8 @@ services:
             - '@ezpublish.api.repository'
             - '@netgen.ezplatform_site.repository.filtering_search_service'
             - '@netgen.ezplatform_site.plugins.field_type.relation_resolver.registry'
+            - '%kernel.debug%'
+            - '@logger'
```

Comment:
Should be `@?logger`, since the service does not have to exist if users turn off logging.

Reply:
Fixed in https://github.com/netgen/ezplatform-site-api/pull/94/commits/f872cc4af9dbaab6e8e8fff10b7c2d8be6893b32